### ISSUE TITLE
fsevents dependency causing mbt build failure

### ISSUE THREAD

**Issue by jbibal-pdc** [SAP-samples/hana-hdbext-promisfied-example#13]
Hi,

Running the `mbt build` command in preparation for deployment to Cloud Foundry fails using SAP Business Application Studio, caused by the **fsevents** module.
![image](https://user-images.githubusercontent.com/105326887/180459981-5eebf370-a058-4f16-a9fb-eca2b68dcc61.png)

Any idea how to bypass this? Or how to resolve this?

Thanks!
John

**Comment by jung-thomas** [comment]
Are you saying that the introduction of the hdbext-promisified module is what causes this error?  The module doesn't use fsevents directly.  This does seem to be common issues with fsevents and several threads suggest removing your package-lock.json and running again. 
https://stackoverflow.com/questions/67987244/unsupported-platform-for-fsevents2-3-2-wanted-osdarwin-archany-cu
https://stackoverflow.com/questions/66093284/unsupported-platform-for-fsevents2-3-2-installation-issue

**Comment by jbibal-pdc** [comment]
Hi @jung-thomas!

![image](https://user-images.githubusercontent.com/105326887/180590764-2a17d3ab-04a2-45a0-8c2d-fe65d3e21acc.png)

Yes but more of indirectly I suppose? It seems like one of the dependencies, as you mentioned is what's causing it. My guess is that it's **mocha** which uses **chokidar**, which then is dependent on **fsevents**.

But yes I did see most of those threads on Stackoverflow. I actually tried everything, and the only thing that worked for me was directly modifying the package-lock.json and changing the:
`"os": [
     "darwin"
],` 

to:
`
"os": [
     "linux"
],
` 

.. as a workaround. I just raised it on here to raise awareness or maybe find a more permanent solution.
